homework_3/src/main/java/com/epam/spring/service/impl/UserServiceImpl.java

```diff
@@ -0,0 +1,53 @@
+package com.epam.spring.service.impl;
+
+import com.epam.spring.controller.dto.UserDto;
+import com.epam.spring.service.UserService;
+import com.epam.spring.service.mapper.UserMapper;
+import com.epam.spring.service.model.User;
+import com.epam.spring.service.repository.UserRepository;
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+@AllArgsConstructor
+public class UserServiceImpl implements UserService {
+
+    private final UserMapper userMapper;
+    private final UserRepository userRepository;
+
+    @Override
+    public UserDto getUser(String email) {
+        return userMapper.mapUserDto(userRepository.getUser(email));
+    }
+
+    @Override
+    public List<UserDto> listUsers() {
+        return userRepository.listUsers().stream().map(userMapper::mapUserDto).collect(Collectors.toList());
```

Comment:
Hi. Let's specify code style rule for methods chain, so it can improve readability. If you are using IntelliJ. Please perform the following steps:
1. Open Settings -> Editor -> Code Style -> Java -> Chained method calls: Wrap always
2. Please press Ctrl - Alt - L
Thank you